catalog/async/catalog-async-data/src/main/java/org/codice/ddf/catalog/async/data/ProcessResourceImpl.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.async.data;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.apache.commons.lang.StringUtils;
+import org.codice.ddf.catalog.async.data.api.internal.ProcessResource;
+
+public class ProcessResourceImpl implements ProcessResource {
+
+    private static final String CONTENT_SCHEME = "content";
+
+    private static final String DEFAULT_NAME = "application/octet-stream";
+
+    private static final String DEFAULT_MIME_TYPE = "content_store_file.bin";
+
+    private static final int UNKNOWN_SIZE = -1;
```

Comment:
❗️ This constant would probably be a good one to put in the interface since it's part of the contract.